docs/appendices/release-notes/5.7.0.rst

```diff
@@ -91,6 +91,9 @@ Scalar and Aggregation Functions
 
 - Added the :ref:`scalar-reverse` scalar function.
 
+- Changed :ref:`scalar-exp` to always return ``double`` type for its result,
+  independently of the input type.
+
```

Comment:
Shouldn't it go the breaking changes section?

Reply:
True, thx for catching!

Reply:
Aren't we treating it as a regular change? https://github.com/crate/crate/pull/15643#discussion_r1510845633

Reply:
I tend to go with breaking actually, @mfussenegger what's your take?

Reply:
or maybe @seut, what do you think?

Reply:
Yes I see this also as a breaking change please move it.
The old behaviour was documented, users may rely on this behaviour. Now that we change it, we should make users aware of it as it may break their existing applications.

Reply:
Thanks all, updated.